pyprojectutils/cli.py

```diff
@@ -488,7 +576,7 @@ def project_parser():
     """Find, parse, and collect project information."""
 
     __author__ = "Shawn Davis <shawn@develmaycare.com>"
-    __date__ = "2016-12-11"
+    __date__ = "2016-12-24"
```

Comment:
Change the date (and version) could create conflicts if others are working on the command. Could we handle this in the release process instead?